src/serialization/sb2_specmap.js

```diff
@@ -1229,6 +1229,21 @@ const specMap = {
             }
         ]
     },
+    '^': {
```

Comment:
Since this block never existed in versions before 3.0, no specmap needed.

Reply:
@apple502j Fixed